### ISSUE TITLE
Pod CPU Alert created with Terraform causes `plugin.downstreamError`

### ISSUE THREAD

**Issue by kevin-journey** [grafana/grafana#62305]
**What happened**: 

We're provisioning alerts with Terraform. One alert, `Pod CPU Utilization` (see JSON and Terraform code below) does get provisioned, but shows as error, and when vieweing it in Grafana, throws the following error:

`
Failed to evaluate queries and expressions: failed to execute conditions: [plugin.downstreamError] failed to query data: error parsing time series query: error unmarshaling query model: json: cannot unmarshal string into Go struct field QueryModel.exemplar of type bool
`

![image](https://user-images.githubusercontent.com/114911328/215080614-d5017e35-2ee5-4807-bec9-1a1629dcc9f2.png)

**What you expected to happen**:

We expected the alert to be provisoned correctly. Other similar alerts such as `Pod Memory Utilizartion` works as expected.

I've compared the JSON of the failing alert with the JSON of the same alert when created manually, and they are exactly the same.

**How to reproduce it (as minimally and precisely as possible)**:

Use the following Terraform code to create the alert:

```
resource "grafana_rule_group" "pod_cpu_utilisation" {

  # Create the Rule group for this rule
  
  name             = "Pod CPU Utilisation"
  folder_uid       = jsondecode(data.jq_query.folderuid.result)
  interval_seconds = 60
  org_id           = 1

  # Start rule creation
  
  rule {
    name           = "${var.environment} - Pod CPU Utilisation"
    for            = "5m"
    condition      = "A"
    no_data_state  = "NoData"
    exec_err_state = "Alerting"
    labels = {
      "environment" = var.environment
      "website" = "true"
      "website_name" = var.customer_namespace
    }

  # Start alert queries
  # First query - From datasource
  
    data {
      ref_id     = "F"
      query_type = ""
      relative_time_range {
        from = 900
        to   = 0
      }
      datasource_uid = var.metrics_ds
      model = jsonencode({
        datasource = {
          type  = "prometheus",
          uid   = var.metrics_ds
        }
        exemplar      = "true"
        expr          = "(sum(rate(container_cpu_usage_seconds_total{namespace=\"${var.customer_namespace}\"}[1m])) by (pod)/sum(kube_pod_container_resource_requests{namespace=\"${var.customer_namespace}\", resource=\"cpu\"}) by (pod))*100"
        hide          = false
        intervalMs    = 15000
        legendFormat  = "{{pod}}"
        maxDataPoints = 43200
        refId         = "F"
      })
    }

  # Second Query - Expression based on data from first query
  
    data {
      ref_id     = "A"
      query_type = ""
      relative_time_range {
        from = 600
        to   = 0
      }
      datasource_uid = "-100"
      model          = <<EOT
{
    "conditions": [
        {
        "evaluator": {
            "params": [
            80
            ],
            "type": "gt"
        },
        "operator": {
            "type": "and"
        },
        "query": {
            "params": [
            "F"
            ]
        },
        "reducer": {
            "params": [],
            "type": "avg"
        },
        "type": "query"
        }
      ],
      "datasource": {
          "type": "__expr__",
          "uid": "-100"
      },
      "expression": "A",
      "hide": false,
      "intervalMs": 1000,
      "maxDataPoints": 43200,
      "refId": "A",
      "type": "classic_conditions"
}
EOT
    }
  }
}
```

**Anything else we need to know?**:

The exact same alert, created through the UI, works correctly.

**Environment**:
- Grafana version: v9.2.2
- Data source type & version: Prometheus / Thanos 11.6.3
- OS Grafana is installed on: Kubernetes

**Comment by zuchka** [comment]
Thanks for creating this issue, @kevin-journey 

That's a pretty informative error message, but I'm curious: have you tried increasing [the verbosity of the Grafana server logs](https://grafana.com/docs/grafana/latest/administration/configuration/#level) to `debug` and noting any errors? For printing to console, [set the console logs to `debug`](https://grafana.com/docs/grafana/latest/administration/configuration/#level-1) as well.  

I tried to reproduce this but I couldn't get the same error. granted I did strip out a lot of the vars and dynamic bits in your file

**Comment by kevin-journey** [comment]
Hi @zuchka 

Thank you for the response. I've managed to fix this thanks to some help on the Grafana Labs Slack channel.

The issue seems to have been caused by:

```
        exemplar      = "true"
```

Changing this to

```
        exemplar      = true
```

Seems to have solved this. The error does not show anymore and the alert works as expected now.

I didn't expect a difference between `"true"` and `true` so never thought of testing to change this.
